server/index.js

```diff
@@ -16,11 +20,17 @@ app.get("/", (req, res) => {
 
 //Error-handling endware
 app.use('/', (err, req, res, next) => {
-	res.status(err.status || 500).send({ message: err.message } || "Internal server error");
+  res.status(err.status || 500).send({ message: err.message } || "Internal server error");
 })
 
 app.listen(PORT, () => {
   console.log(`App is listening at localhost:${PORT}`);
 });
-
-module.exports = app
\ No newline at end of file
+//May add conditions to { force: true } later to avoid delating the entire database in the deployed app
+db.sync({ force: true })
+  .then(() => {
+    console.log('db synced')
+  })
+  .catch(error => console.log(error))
```

Comment:
We may want to consider doing more than simply logging an error.

If this happens in production (e.g. your deployed site) - what do you want to happen?

Reply:
Thanks Eliot. What if we make it so that the app starts listening after db sync. If db fails to sync, then we send an error page to the client?

Reply:
That sounds like a better solution than what is presently happening! @nries1